test/drop.test.js

```diff
@@ -0,0 +1,13 @@
+const { drop } = require("../lib");
+
+test("Should return whole array if count is zero", () => {
+  expect(drop([1,2,3],0)).toEqual([1,2,3]);
+});
+
+test("Should return without first N elements if array length is >= N", () => {
+  expect(drop([1,2,3],2)).toEqual([2]);
```

Comment:
Should this result in [3] since 1 and 2 get dropped?

Reply:
Yes, it should return [3]